packages/core-typings/src/ILivechatVisitor.ts

```diff
@@ -45,7 +45,10 @@ export interface ILivechatVisitor extends IRocketChatRecord {
 	};
 	livechatData?: ILivechatData;
 	contactManager?: {
-		username: string;
+		_id?: string;
+		username?: string;
+		name?: string;
+		emails?: { address: string }[];
```

Comment:
`contactManager` can be `undefined`, but all of it's properties can also be `undefined`? 
This seem to differ from the BE implementation.
```suggestion
		_id: string;
		username: string;
		name: string;
		emails: { address: string }[];
```